### PR TITLE
updating libclang binding for jextract changes and llvm 13.0.0

### PR DESCRIPTION
**Pull request by sundararajana**
regenerating LibClang binding for jextract changes and llvm 13.0.0

---------
### Progress
- [x] Change must not contain extraneous whitespace
- [x] Change must be properly reviewed (no review required)

### Reviewers
 * [Maurizio Cimadamore](https://openjdk.org/census#mcimadamore) (@mcimadamore - Committer)

### Reviewing
<details><summary>Using <code>git</code></summary>

Checkout this PR locally: \
`$ git fetch https://git.openjdk.org/jextract pull/52/head:pull/52` \
`$ git checkout pull/52`

Update a local copy of the PR: \
`$ git checkout pull/52` \
`$ git pull https://git.openjdk.org/jextract pull/52/head`

</details>
<details><summary>Using Skara CLI tools</summary>

Checkout this PR locally: \
`$ git pr checkout 52`

View PR using the GUI difftool: \
`$ git pr show -t 52`

</details>
<details><summary>Using diff file</summary>

Download this PR as a diff file: \
<a href="https://git.openjdk.org/jextract/pull/52.diff">https://git.openjdk.org/jextract/pull/52.diff</a>

</details>

### CODE REVIEW

File: src/main/java/org/openjdk/jextract/clang/libclang/RuntimeHelper.java [hunk 1]
```diff
@@ -62,11 +62,9 @@ private RuntimeHelper() {}
             (size, align) -> MemorySegment.allocateNative(size, align, MemorySession.openImplicit());
 
     static {
-        // Manual change to handle platform specific library name difference
-        String libName = System.getProperty("os.name").startsWith("Windows")? "libclang" : "clang";
-        System.loadLibrary(libName);
-
-        SYMBOL_LOOKUP = name -> SymbolLookup.loaderLookup().lookup(name).or(() -> LINKER.defaultLookup().lookup(name));
+        System.loadLibrary("clang");
```
Comment (JornVernee) [inline src/main/java/org/openjdk/jextract/clang/libclang/RuntimeHelper.java:65]: This needs the fix for Windows

Comment (sundararajana) [reply src/main/java/org/openjdk/jextract/clang/libclang/RuntimeHelper.java:65]: oops. missed that manual change. will fix it.

**Review by mcimadamore — APPROVED** [review]
Looks good

**Pull request closed without merging**